build-multi.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-export VERSION=101
+export VERSION=Brave-1.0
```

Comment:
Hm, should we remove VERSION here and just use BRAVEVERSION?
The image tag can probably be more descriptive, like `brave-BRAVEVERSION` in case we add other browsers in the future..
This is just for local builds though, but should be consistent.